backend/src/main/resources/db/changelog/db.changelog-master.yaml

```diff
@@ -1715,6 +1715,25 @@ databaseChangeLog:
               migrations_user_id updated_by
             FROM ${database.defaultSchemaName}.person CROSS JOIN migration_user
             WHERE test_result_delivery IS NOT NULL;
+  - changeSet:
+      id: add-test-length
+      author: emma@skylight.digital
+      comment: Add test length column to the device type table. Test length will be based on the model used.
+      validCheckSum: # hopefully this is temporary because I have no idea why the checksum changed. Putting it in so I can test changes
```

Comment:
This happens when you have a migration and you try to change it and re-apply it. This probably indicates that the migration's most recent changes didn't get run—I usually work around this by snapshotting and restoring repeatedly while I'm developing migrations, but if you don't have a snapshot it may be a little more tiresome to re-test.

Reply:
Resolved offline, but for posterity:

If you don't have a snapshot, you can roll back your migration and re-apply it. This can be done using `psql` (in your dev environment!) with the following commands:
`delete from simple_report.databasechangelog where id = '$MY-MIGRATION-ID';`
`alter table $EDITED_TABLE drop column $MY-NEW-COLUMN;`

Then re-start the app, and your migration's changes should be run as well.